src/legacy/core_plugins/metrics/public/components/lib/tick_formatter.js

```diff
@@ -18,36 +18,61 @@
  */
 
 import handlebars from 'handlebars/dist/handlebars';
-import { durationInputOptions } from './durations';
-import { capitalize, isNumber } from 'lodash';
+import { isNumber } from 'lodash';
 import { fieldFormats } from 'ui/registry/field_formats';
 
-const durationsLookup = durationInputOptions.reduce((acc, row) => {
-  acc[row.value] = row.label;
-  return acc;
-}, {});
+const inputFormat = {
+  'ps': 'picoseconds',
+  'ns': 'nanoseconds',
+  'us': 'microseconds',
+  'ms': 'milliseconds',
+  's': 'seconds',
+  'm': 'minutes',
+  'h': 'hours',
+  'd': 'days',
+  'w': 'weeks',
+  'M': 'months',
+  'Y': 'years',
+};
+
+const outputFormat = {
+  'humanize': 'humanize',
+  'ms': 'asMilliseconds',
+  's': 'asSeconds',
+  'm': 'asMinutes',
+  'h': 'asHours',
+  'd': 'asDays',
+  'w': 'asWeeks',
+  'M': 'asMonths',
+  'Y': 'asYears',
+};
+
+const isDuration = (format) => {
+  const input = Object.keys(inputFormat).join('');
+  const output = Object.keys(outputFormat).join('');
```

Comment:
I think we can move these two `const` and the RegExp out from this function, we don't need to reinitialize the input the output and the regexp for every isDuration request